### ISSUE TITLE
Add term breath-holding spells

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10498]
**Preferred term label:**
Breath-holding spell

**Synonyms**

**Definition (free text, please give PubMed ID)**
A benign paroxysmal nonepileptic disorder occurring in healthy children 6 to 48 months of age. The episodes start with a provocation such as emotional upset or minor injury, and might progress to breath holding, cyanosis, and syncope. PMID: 1736640

**Parent term (use hpo.jax.org/app)**
Abnormal pattern of respiration HP:0002793

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, Beare Stevenson cutis gyrata syndrome, omim123790 (PMID12181710). 
As stated in the definition, it can occur in healthy children (Almost 5% of the pediatric population, PMID: 9225356)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000950
- label: Breath-holding spell
- parents: [Abnormal pattern of respiration (HP:0002793)](https://hpo.jax.org/app/browse/term/HP:0002793)
- pmid: PMID:1736640
- def: A benign paroxysmal nonepileptic disorder occurring in healthy children 6 to 48 months of age. The episodes start with a provocation such as emotional upset or minor injury, and might progress to breath holding, cyanosis, and syncope.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10498
- synonyms:
